pom.xml

```diff
@@ -179,9 +179,9 @@
     <version.org.glassfish.jaxb>2.3.2</version.org.glassfish.jaxb>
     <version.org.gwtbootstrap3>1.0.1</version.org.gwtbootstrap3>
     <version.org.gwtbootstrap3-extras>1.0.2</version.org.gwtbootstrap3-extras>
-    <version.org.hibernate>5.3.7.Final</version.org.hibernate>
+    <version.org.hibernate>5.3.15.Final</version.org.hibernate>
```

Comment:
For Oracle 19, we need at least 5.3.16, due to a bug to resolve dialect:
https://hibernate.atlassian.net/browse/HHH-13184
Fixed at 5.3.16
https://hibernate.atlassian.net/secure/ReleaseNote.jspa?version=31822&styleName=Html&projectId=10031